README.md

```diff
@@ -340,20 +339,20 @@ export interface TrackableStoreDelta<T> extends StoreDelta<T> {
 
 ```
 
-As with the observable store mixin, the locally tracked data in a `MappedQueryTransformResult` has the potential to become out of sync with the underlying storage. If the source `ObservableStore` has `fetchAroundUpdates` set to true, then any query transform results produced from it will only send up to date information to observers. If the source is not fetching around updates, the `track()` method provided as part of the `MappedQueryTransformResult` interface can be used to create a copy of a `QueryTransformResult` that will fetch after any updates from its source to make sure it has the latest data. `track()` produces a `TrackedQueryTransformResult`, which has a `release()` method that provides a new, non-tracked query transform result.
+As with the observable store mixin, the locally tracked data in a `MappedQueryResult` has the potential to become out of sync with the underlying storage. If the source `ObservableStore` has `fetchAroundUpdates` set to true, then any query transform results produced from it will only send up to date information to observers. If the source is not fetching around updates, the `track()` method provided as part of the `MappedQueryResult` interface can be used to create a copy of a `QueryResult` that will fetch after any updates from its source to make sure it has the latest data. `track()` produces a `TrackedQueryResult`, which has a `release()` method that provides a new, non-tracked query transform result.
```

Comment:
Is Observable store still a mixin, I thought it was now an interface, so perhaps change As with the observable store mixin to As with the `ObservableStoreInterface`